core-library/src/main/java/org/silverpeas/core/admin/service/GroupSynchronizationRule.java

```diff
@@ -452,10 +434,9 @@ private static String getRuleExpression(
       DomainDriverManager domainDriverManager =
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "domainDriverManager" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1481)